### PR TITLE
Feature/pull config docs into read me

### PR DESCRIPTION
**Pull request by jimmypuckett**
@ssfinney I pulled the config's documents to the ready to make it more clear what the system does.  When you get a chance, can you review it.

Thanks

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -46,17 +46,76 @@ The command is registered with laravel as ```garbageman:purge```.  You can run i
...
+### Fire purge events (fire_purge_events)
+Allow hook into the purge of each record by throwing events before & after deleting of each record. There are 2 events 
+thrown:
+
+* garbageman.purging:<full/model/name>
+* garbageman.purged:<full/model/name>
+
+The model is passed with each of the events. The "purging" event is thrown just *before* the actual delete & "purged" is thrown just *after* the actual delete.
+
+This is an expensive operation as it requires a SQL command for each record to delete so that the record can be thrown with the events. Therefore, unless you need to catch the events to preform some other action, leave this false to allow all records per model to get deleted with a single SQL call.
```
Comment (ssfinney) [inline readme.md:80]: Instead of "leave this false", do you mean to leave it blank/null?

**Comment by ssfinney** [comment]
@jimmypuckett I've pushed a few changes of my own for you to look at, and a comment for you to review.

**Comment by jimmypuckett** [comment]
@ssfinney I made one small tweak & I think that it is good to go when you are.

File: readme.md [hunk 2]
```diff
@@ -46,17 +45,73 @@ The command is registered with laravel as ```garbageman:purge```.  You can run i
...
+
+### Fire purge events (fire\_purge\_events)
+Allow hook into the purge of each record by throwing events before & after deleting of each record. There are 2 events thrown:
+
+* garbageman.purging:\<full/model/name\>
+* garbageman.purged:\<full/model/name\>
+
+The model is passed with each of the events. The "purging" event is thrown just *before* the actual delete & "purged" is thrown just *after* the actual delete.
+
+This is an expensive operation as it requires a SQL command for each record to delete so that the record can be thrown with the events. Therefore, unless you need to catch the events to preform some other action, leave this false to allow all records per model to get deleted with a single SQL call.
```
Comment (ssfinney) [inline readme.md:80]: ~~@jimmypuckett Instead of "leave this false", do you mean to leave it blank/null?~~

Nevermind

**Pull request merged**